scripts/deploy.sh

```diff
@@ -9,11 +9,17 @@ TARGET=$3
 COMMIT=$4
 BRANCH=$5
 IS_FORKED_PR=$6
-NPM_SCRIPT="yarn run deploy-$TARGET"
 REPO_ROOT=`dirname "$0"`/..
 BASE_DOCKER_COMPOSE=$REPO_ROOT/docker/docker-compose.yml
 DOCKER_COMPOSE_FILE=$REPO_ROOT/docker/docker-compose.ci.yml
 
+if [ "$SERVICE" = "unlock-protocol-com" ] ; then
```

Comment:
We really need to rename that service. Wdyt? In a later PR maybe?

Reply:
this exists because the folder name is `unlock-protocol.com` but the docker service name is `unlock-protocol-com`. Another option will be to hyphenate all names by replacing special chars by dashes.

Reply:
Let's rename the folder in a later PR